python/ray/_private/runtime_env/packaging.py

```diff
@@ -352,9 +457,15 @@ def _store_package_in_gcs(
     return len(data)
 
 
-def _get_local_path(base_directory: str, pkg_uri: str) -> str:
-    _, pkg_name = parse_uri(pkg_uri)
-    return os.path.join(base_directory, pkg_name)
+def _get_local_path(base_directory: str, pkg_uri: str) -> Tuple[str, UriType]:
+    _, uri_type, pkg_name, path = parse_uri(pkg_uri)
+    if uri_type is not UriType.REMOTE:
+        # Support relative path for tests.
+        RELATIVE_PATH_PREFIX = "/./"
```

Comment:
I didn't understand this part, would you mind adding more detail in the code comment?  Is it just because `/./` is required in the C++ tests for some reason?